src/datablock/DatablockFunctions.h

```diff
@@ -5,5 +5,6 @@ namespace UKControllerPlugin {
 
         int ConvertAltitudeToFlightLevel(int altitude);
         int ConvertFlightLevelToAltitude(int flightLevel);
+        int NormaliseFlightLevelFromString(std::string level);

```

Comment:
```suggestion
        int NormaliseFlightLevelFromString(const std::string & level);

```

Reply:
Done!

Reply:
A const reference doesn't work here due to the use of `std::transform` in the function itself